markdown/dev/reference/api/part/draft/en.md

```diff
@@ -45,3 +45,11 @@ access the following properties:
 | `Bezier`          | The [bezier-js](https://pomax.github.io/bezierjs/) library's `Bezier` named export |
 || **_Return value_**   |
 | `part`            | Your draft method **must** return this |
+
```

Comment:
I find this adds more confusion than anything else.
There is nowhere ever that it's mentioned that there's an `optionalMeasurements` key. 

Just because somebody made a mistake once and then others blindly copy/pasted the same mistake, I don't think we need to add this to the docs.

Reply:
I don't know if the addition to the docs is absolutely needed. However, I think that it helps to add clarity, and it does not add confusion.

In #3933, before learning that `optionalMeasurements` did not exist, I was convinced that it did actually exist and was somehow missing from our code. I believed it to the point that I spent time working on a code "fix" to add `optionalMeasurements` back into part shorthand to try to resolve the problem.